parsing/location.ml

```diff
@@ -250,6 +251,110 @@ let get_pos_info pos =
   (pos.pos_fname, pos.pos_lnum, pos.pos_cnum - pos.pos_bol)
 ;;
 
+(* basic color handling *)
+module Color = struct
```

Comment:
Thinking twice: the `styles` stuff could remain inside `Color` if we consider that it should centralize all color styles in the compiler distribution (and happens to only contain location/error/warning stuff for now). Having the information that is currently in `default_styles` centralized in the same place may make it easier for people to customize default colors in the future. So regarding the location of the `styles` stuff, I would accept whatever @c-cube feels is best.